src/index.scss

```diff
@@ -28,6 +28,18 @@ h5 {
   font-family: HelsinkiGrotesk-Bold, Arial, sans-serif;
 }
 
+fieldset {
```

Comment:
These are set to try to reset the default css values for fieldset that it gets from browsers.

Reply:
I'm not sure about the browser support of CSS logical properties 🤔

Reply:
Good catch, maybe we could just replace those with the regular margin-right and margin-left?